helx/Chart.lock

```diff
@@ -4,7 +4,7 @@ dependencies:
   version: 0.1.9
```

Comment:
When I pulled and ran `helm dependency update`, some of the versions in this file (like this one) changed. Seems it may need to be regenerated

Reply:
I'll recreate this merge.  At the last minute I pulled in some changes from Murali's k8s API updates.  Probably those changes.